modules/olm-installing-from-operatorhub-using-web-console.adoc

```diff
@@ -77,8 +77,10 @@ endif::[]
 ifdef::olm-user[]
 .. Choose a specific, single namespace in which to install the Operator. The Operator will only watch and be made available for use in this single namespace.
 endif::[]
-
-.. If the cluster is in STS mode, add the role ARN in the *role ARN* field.
+.. If the cluster is in AWS STS mode, enter the Amazon Resource Name (ARN) of the AWS IAM role of your service account in the *role ARN* field.
++
+image::oadp-install-operator-role-arn.png[Entering the ARN]
+To create the role's ARN, follow the procedure described in step 2 of link:https://access.redhat.com/documentation/en-us/red_hat_openshift_service_on_aws/4/html/tutorials/cloud-experts-deploy-api-data-protection#prepare-aws-account_cloud-experts-deploy-api-data-protection[Preparing AWS account].
```

Comment:
We usually try to avoid calling out specific steps because they could be updated or changed later and might create a issue if its not referenced correctly 
```suggestion
To create the role's ARN, follow the procedure described in link:https://access.redhat.com/documentation/en-us/red_hat_openshift_service_on_aws/4/html/tutorials/cloud-experts-deploy-api-data-protection#prepare-aws-account_cloud-experts-deploy-api-data-protection[Preparing AWS account].
```